smac/utils/io/cmd_reader.py

```diff
@@ -10,16 +19,146 @@
 __email__ = "lindauer@cs.uni-freiburg.de"
 __version__ = "0.0.1"
 
+in_reader = InputReader()
+parsed_scen_args = {}
+logger = None
+
+def truthy(x):
+    if isinstance(x, bool):
+        return x
+    elif isinstance(x, int) or isinstance(x, float):
+        return x != 0
+    elif isinstance(x, str):
+        return bool(distutils.util.strtobool(x))
+    else:
+        return False
+
+
+class CheckScenarioFileAction(Action):
```

Comment:
Could you please add brief docstrings for all classes (also in the following)

Reply:
Initially left them out because the class name is the documentation. Nevertheless added docstrings.